libdino/src/entity/file_transfer.vala

```diff
@@ -53,6 +53,7 @@ public class FileTransfer : Object {
     }
     public string path { get; set; }
     public string? mime_type { get; set; }
+    // TODO(hrxi): expand to 64 bit
```

Comment:
Just do it.

**Not** blocking for merging.

Reply:
Not yet done, required some changes in database code but the database code didn't like `int64`.